src/main/java/frc/robot/commands/ChargeStationBalance.java

```diff
@@ -7,11 +7,18 @@
 import edu.wpi.first.math.kinematics.ChassisSpeeds;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import edu.wpi.first.wpilibj.Timer;
-import frc.robot.Constants;
 import frc.robot.subsystems.DriveTrain;
+import edu.wpi.first.math.geometry.Rotation2d;
 
 public class ChargeStationBalance extends CommandBase {
 
+  //private final Rotation2d m_rampAngle = Rotation2d.fromDegrees(-10); //useful if we are going to automate going on the ramp
```

Comment:
I saw you moved these constants here from Constants.java? Did one the coaches ask you to do so? because I think they should be in Constants.java and the variable names don't need "m_" and should all be capitalized.